packages/storybook/stories/Eyebrow.stories.tsx

```diff
@@ -0,0 +1,25 @@
+import { StoryDocs } from './wc-helpers';
+
+export default {
+  title: 'Components/Eyebrow',
+  id: 'components/eyebrow',
+  parameters: {
+    componentSubtitle: 'Eyebrow component class',
+    docs: {
+      page: () => (
+        <StoryDocs storyDefault={Default} componentName="Eyebrow" />
+      ),
+    },
+  }
+};
+
+const Template = ({}) => (
+  <h1>
+    <span className="vads-c-eyebrow">
+      Eyebrow Title
+    </span>
+    Main Headline
+  </h1>
```

Comment:
I converted this Storybook file from markdown to `.tsx` so that we could provide the code block example.

**before**

![Screenshot 2025-04-09 at 7 59 08 AM](https://github.com/user-attachments/assets/988f2a56-6b41-43c8-9031-06e9ea7d07dc)

**after**

![Screenshot 2025-04-09 at 7 59 14 AM](https://github.com/user-attachments/assets/93df6a77-b980-4351-bc40-9fc80eca1182)